frontend/chandragupta-ii-explorer/index.html

```
@@ -0,0 +1,458 @@
+                    <li><strong>Legge, James (1886)</strong>: <em>A Record of Buddhistic Kingdoms (Faxian's Travelogue Translation)</em>, Clarendon Press.</li>
+                    <li><strong>Archaeological Survey of India (ASI)</strong>: Mehrauli Iron Pillar &amp; Mathura Inscription Reports.</li>
+                </ul>
+            </section>
+        </div>
+    </main>
+
+    <!-- Interactive Image Modal Lightbox -->
+    <div id="img-modal-lightbox" class="img-modal" role="dialog" aria-hidden="true" aria-label="Interactive Image Lightbox">
+        <div class="img-modal-content">
+            <button type="button" class="img-modal-close" id="img-modal-close" aria-label="Close Lightbox">&times;</button>
+            <img id="img-modal-target" src="" alt="Enlarged historical image">
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)